utils/redirect_url.ts

```diff
@@ -0,0 +1,44 @@
+import { EmailString } from "@pagopa/ts-commons/lib/strings";
+import { ValidUrl } from "@pagopa/ts-commons/lib/url";
+import base64url from "base64url";
+import { TokenQueryParam } from "./middleware";
+import { ValidationErrors } from "./validation_errors";
+
+/**
+ * Returns a ValidUrl that represents a successful validation
+ */
+export const confirmChoicePageUrl = (
+  url: ValidUrl,
+  token: TokenQueryParam,
+  email: EmailString
+): ValidUrl =>
+  ({
+    href: `${url.href}?token=${token}&email=${base64url(email)}`
+  } as ValidUrl);
+
+/**
+ * Returns a ValidUrl that represents a successful validation
+ */
+export const validationSuccessUrl = (
+  validationCallbackUrl: ValidUrl,
+  timeStampGenerator: () => number
```

Comment:
This can be removed, this information is not useful for monitoring. The logs already have a TimeGenerated value that refers the time when the URL is requested.

Reply:
removed in https://github.com/pagopa/io-functions-public/pull/72/commits/7d115d231be6393d3b76e2db3ca557f7419a4eab